pages/integrations/sso/okta.md

```diff
@@ -43,7 +43,7 @@ Go to your Buildkite application in Okta to set up deprovisioning:
 
 ### Provisioning existing users
 
-Existing Okta users aren't automatically provisioned in Buildkite; you'll need to sync your users in order to deprovision them.
+Existing Okta users aren't automatically provisioned in Buildkite; you'll need to sync your users in order to deprovision them. Users will have their Buildkite accounts created via [JIT provisioning](/docs/integrations/sso#frequently-asked-questions-do-you-support-jit-provisioning).
```

Comment:
This reads better to me, but I'm not sure if it's correct. Can you check this conveys the same meaning, or let me know what's not right?

```suggestion
Buildkite creates accounts for existing Okta users with just-in-time user provisioning (JIT provisioning). To deprovision users, you need to sync them.
```

Reply:
@mbelton-buildkite These are much better words! Thanks for checking :)